routes/blogy.js

```diff
@@ -0,0 +1,77 @@
+const router = require("express").Router();
+const blog = require("../backend_model/blog_post");
+const blogcomment = require("../backend_model/Comment_post");
+
+// For blog post
+router.post("/blog", async (req, res) => {
+  const { title, content, author } = req.body;
+  console.info(req.body);
+  const newBlog = new blog({ title, content, author });
+
+//   try and catch
+  try {
+      await newBlog.save()
+      res.send("blog saved")
+  } catch (err) {
+      res.json({ error: err })
+      console.log({ error: err })
+  }
+});
+
+// to get all blog
+router.get("/blog", async (req, res) => {
+    try {
+       await blog.find()
+       res.send("blogs found!")
+    } catch (err) {
+        res.json({ error: err })
+        console.log({ error: err })
+    }
+});
+
+
+// Get blog by id
+router.get("/blog/:id/blogcomment", async (req, res) => {
```

Comment:
Update this route path accordingly.

https://apiguide.readthedocs.io/en/latest/build_and_publish/use_RESTful_urls.html